docs/pages/javascript.md

```diff
@@ -28,99 +28,145 @@
 Stable releases are published on https://www.npmjs.com/package/zenroom that
 have a slow pace release schedule that you can install with
 
+<!-- tabs:start -->
+
+### ** npm **
+
 ```bash
-yarn add zenroom
-# or if you use npm
 npm install zenroom
 ```
 
+### ** yarn **
+
+```bash
+yarn add zenroom
+```
+
+### ** pnpm **
+
+```bash
+pnpm add zenroom
+```
+
+<!-- tabs:end -->
 
 * * *
 
 ## 🎮 Usage
 
-The binding consists of one main function::
+The binding consists of two main functions:
 
-**zencode_exec** to execute [Zencode](https://dev.zenroom.org/#/pages/zencode-intro?id=smart-contracts-in-human-language). To learn more about zencode syntax look [here](https://dev.zenroom.org/#/pages/zencode-cookbook-intro)
+- **zencode_exec** to execute [Zencode](https://dev.zenroom.org/#/pages/zencode-intro?id=smart-contracts-in-human-language). To learn more about zencode syntax look [here](https://dev.zenroom.org/#/pages/zencode-cookbook-intro)
+- **zenroom_exec** to execute our special flavor of Lua enhanced with Zenroom's [special effects](https://dev.zenroom.org/#/pages/lua)
```

Comment:
This was intentionally left out, becasue @jaromil don't really like people using lua... bho... maybe we want to tell about the new introspection, though...

Reply:
I saw that we have it on the [npm readme](https://www.npmjs.com/package/zenroom), thus I added it also here.

For the introspection and other APIs I listed them with no example at the end, thinking they were not the main part, but I can expand them with example if we want 🌻